src/PlutotvData.h

```diff
@@ -74,6 +74,7 @@ class ATTR_DLL_LOCAL PlutotvData : public kodi::addon::CAddonBase,
 
   std::string GetChannelStreamURL(int uniqueId);
   std::string GetSettingsUUID(const std::string& setting);
+  int GetSettingsStartChannel();
```

Comment:
This method most probably can be `const`. => ` int GetSettingsStartChannel() const;`